static/js/chart/draw.ts

```diff
@@ -130,7 +126,7 @@ function wrap(
           .append("tspan")
           .attr("x", 0)
           .attr("y", y)
-          .attr("dy", ++lineNumber * lineHeight + dy + "em")
```

Comment:
This was added for long text x-axis ticks. Do the charts on the place pages and dev page look ok when set to narrow widths?

Reply:
revert this back. I need to add attr("y") for the legend text to work with this